extensions/import/src/wizard/pages/deployPlanPage.ts

```diff
@@ -0,0 +1,288 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+import * as sqlops from 'sqlops';
+import * as nls from 'vscode-nls';
+import * as parser from 'htmlparser2';
+import { DacFxDataModel } from '../api/models';
+import { DataTierApplicationWizard } from '../dataTierApplicationWizard';
+import { DacFxConfigPage } from '../api/dacFxConfigPage';
+
+const localize = nls.loadMessageBundle();
+
+export enum deployPlanXmlTag {
+	Alert = 'Alert',
+	Operation = 'Operation',
+	Item = 'Item'
+}
+
+export enum attributeName {
```

Comment:
Can this enum be moved into the deployPlanXmlTag type?

Reply:
I had these separate since these are attributes, not xml tags. I can move them together if that's more clear though.

Reply:
I think a type for attributes and another for elements is going too far. I think one type can be used for both. I was thinking:
````
export enum DeployPlanXml {
   AlertElement = 'Alert',
   NameAttribute = 'Name',
   ... 
}
````
This makes clear where the xml tags/attributes are referenced in code, and provides some semantic meaning.